app/javascript/react/components/App.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { Router, browserHistory, Route, IndexRoute } from 'react-router';
 
 import AirportContainer from '../containers/AirportContainer'
+import NewAirportFormContainer from '../containers/NewAirportFormContainer'
 
 export const App = props => {
   return(
     <div>
       <Router history={browserHistory}>
         <Route path='/airports' component={AirportContainer} >
         </Route>
+        <Route path='/airports/new' component={NewAirportFormContainer}/>
```

Comment:
Your first route uses a </Route> your second route is self closing. You should pick a convention and stick with it team wide.

Reply:
Going with the self-closing -resolved